pkg/controller/node/node_controller.go

```diff
@@ -459,16 +459,12 @@ func (ctrl *Controller) syncMachineConfigPool(key string) error {
 		return err
 	}
 
-	progress, err := makeProgress(pool, nodes)
+	maxunavail, err := maxUnavailable(pool, nodes)
```

Comment:
there's something that we might need to account tho. The `master` cannot really have more than 1 node updating at a time (etcd-quorum-guard). We should add a check before GA to prevent people to set that to >1 (or just ignore if it's >1 and always rolls 1 by 1)

Reply:
Have an update locally for this, will push after CI results come in for the current commit.

Reply:
done :arrow_down: